.github/workflows/refresh-staging.yml

```
@@ -0,0 +1,57 @@
+
+  # Allow manual triggering
+  workflow_dispatch:
+
+jobs:
+  refresh-staging:
+    runs-on: ubuntu-latest
+    name: Export from Prod & Seed Staging
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)